fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1263,6 +1267,12 @@ alterLoadStatement
         jobProperties?
     ;
 
+// ------------------------------------------- Compaction Statement ----------------------------------------------------------
+
+cancelCompactionStatement
+    : CANCEL COMPACTION (WHERE expression)?
+    ;
```

Comment:
If where expression is necessary, can we directly use `CANCEL COMPACTION WHERE expression` ?